tools/python/upload_and_run_browserstack_tests.py

```diff
@@ -169,4 +169,5 @@ def build_query_loop(build_id, test_platform, id, token):
     print("Test suite details: ", test_suite_details_url)
     print("=" * 30)
     if tests_status != "passed":
-        raise Exception(f"Tests failed. Go to {test_suite_details_url} for more details.")
+        output_str = f"Tests failed. Go to {test_suite_details_url} for more details & go to 'End-to-end tests for iOS & Android ' in the ORT Mobile OneNote to see more details on how to access the BrowserStack logs."
```

Comment:
good idea to reference the onenote page.

maybe it can be more explicit about how to locate it in OneNote. e.g., "ONNX Ecosystem Team Notebook" > "ORT Mobile" > "End-to-end tests for iOS & Android".

also, please try to keep line lengths to 120 characters.

Reply:
Updated with more detailed instructions and broke up into smaller lines -- which will also make the message more readable in the pipelines I think